checkov/terraform/checks/resource/aws/PasswordPolicyExpiration.py

```diff
@@ -19,7 +20,7 @@ def scan_resource_conf(self, conf):
         """
         key = 'max_password_age'
         if key in conf.keys():
-            if conf[key][0] >= 90:
+            if force_int(conf[key][0]) >= 90:
```

Comment:
@arielkru What happens if the value isn't a number, but a string that points to a variable?